packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -578,13 +603,7 @@ const BaseScrollerItem = ({
   }, [dividerType, post, unreadCount, showDayDivider]);
 
   return (
-    <View
-      onLayout={handleLayout}
-      {...useMemo(
-        () => (channelType === 'gallery' ? { aspectRatio: 1, flex: 0.5 } : {}),
```

Comment:
nb: removed the conditional `flex: 0.5` here – I think using any `flex: <number>` is equivalent, since we just want all of the items in the grid to fill equally